src/HappyEyeBallsConnectionBuilder.php

```diff
@@ -82,7 +81,11 @@ public function connect()
                 };
             };
 
-            $that->resolverPromises[Message::TYPE_AAAA] = $that->resolve(Message::TYPE_AAAA, $reject)->then($lookupResolve(Message::TYPE_AAAA));
+            $lookupResolveError = function ($type) {
+                return function ($error) { };
+            };
+
+            $that->resolverPromises[Message::TYPE_AAAA] = $that->resolve(Message::TYPE_AAAA, $reject)->then($lookupResolve(Message::TYPE_AAAA), $lookupResolveError(Message::TYPE_AAAA));
```

Comment:
Diff should be unneeded as the `resolve()` method always resolves and never rejects.